plugin-testlib/build.gradle.kts

```diff
@@ -34,5 +35,6 @@ dependencies {
     api(gradleApi())
     api(gradleTestKit())
     api(TestLib.lib)
+    api("${JUnit.Jupiter.api}:${JUnit.version}")
```

Comment:
Could we have something like `JUnit.Jupiter.api.artifact` extension which would declare the specified library as an artifact in the hard-coded version (in this case `JUnit.version`)?